.travis.yml

```diff
@@ -26,13 +26,15 @@ before_install:
   - composer self-update
   - if [ "$DEPENDENCIES" = "dev" ]; then perl -pi -e 's/^}$/,"minimum-stability":"dev"}/' composer.json; fi;
   - if [[ ! $skip && ! $PHP = hhvm* ]]; then phpenv config-rm xdebug.ini; fi
-  - if [ "$SYMFONY_VERSION" != "" ]; then composer require --no-update symfony/framework-bundle:"$SYMFONY_VERSION"; fi
-  - if [ "$SYMFONY_VERSION" != "" ]; then composer require --no-update symfony/console:"$SYMFONY_VERSION"; fi
+  - if [ "$SYMFONY_VERSION" != "" ]; then composer require --no-update symfony/framework-bundle:"$SYMFONY_VERSION" symfony/console:"$SYMFONY_VERSION"; fi
+
+install:
+  - composer update $COMPOSER_FLAGS
 
 before_script:
   - sh -c 'if [ "$LIBRABBITMQ_VERSION" != "" ]; then ./Tests/bin/install_librabbitmq.sh; fi;'
   - sh -c 'if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then echo "extension=amqp.so" >> `php --ini | grep "Loaded Configuration" | sed -e "s|.*:\s*||"`; fi;'
   - sh -c "sudo ./Tests/bin/prepare_rabbit.sh"
 
-install:
-  - composer update $COMPOSER_FLAGS
+# Due to not yet solved issue with phpdocumentor & php 5.5 https://github.com/phpDocumentor/ReflectionDocBlock/issues/80
+script: vendor/phpunit/phpunit/phpunit
```

Comment:
why not using ``vendor/bin/phpunit`` instead, which is what gets exposed by Composer (and so robust in case PHPUnit moves the file inside their package, as they did in the past)